packages/aws-rfdk/bin/stage-deadline.ts

```diff
@@ -56,88 +57,135 @@ while (n < args.length) {
   n++;
 }
 
-// Automatically populate the installer & recipe URI using the version, if it is provided.
-if (deadlineReleaseVersion !== '') {
-  try {
-    const version = Version.parse(deadlineReleaseVersion);
-    if(version.isLessThan(Version.MINIMUM_SUPPORTED_DEADLINE_VERSION)) {
-      console.error(`ERROR: Unsupported Deadline Version ${version.toString()}. Minimum supported version is ${Version.MINIMUM_SUPPORTED_DEADLINE_VERSION} \n`);
-      usage(1);
-    }
-  } catch(e) {
-    console.error(`ERROR: ${(e as Error).message} \n`);
-    usage(1);
-  }
+if (!fs.existsSync(outputFolder)) {
+  fs.mkdirSync(outputFolder);
+} else if (fs.readdirSync(outputFolder).length > 0) {
+  console.error('The target directory is not empty.');
+  process.exit(1);
+}
 
-  // populate installer URI
-  if (deadlineInstallerURI && deadlineInstallerURI.length > 0) {
-    console.info('INFO: Since deadline release version is provided, "deadlineInstallerURI" will be ignored.');
-  }
-  deadlineInstallerURI = `s3://thinkbox-installers/Deadline/${deadlineReleaseVersion}/Linux/DeadlineClient-${deadlineReleaseVersion}-linux-x64-installer.run`;
+const handler = new VersionProvider();
```

Comment:
It's really weird for us to stand up a `SimpleCustomResource` here in this script, outside of a CDK construct. Could we refactor the code so that the methods like `getUrisForPlatform` and `getLatestVersion` are outside of the `VersionProvider` class? That way you can use them here without standing up the full `VersionProvider` and having to call its `doCreate` method.

I think it should be able to be accomplished by moving the methods from `VersionProvider` in `handler.ts` into another file inside the `version-provider` directory and then move their tests to a separate file as well. Then in this script you'd do something similar to what `doCreate()` is doing.